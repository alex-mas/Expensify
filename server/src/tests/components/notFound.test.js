import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import ErrorNotFound from '../../components/notFound.js';


describe('ErrorNotFound component unit tests', function(){
    test('should render ErrorNotFound correctly',()=>{
        const wrapper = shallow(<ErrorNotFound/>);
        expect(wrapper).toMatchSnapshot();
    });
});
