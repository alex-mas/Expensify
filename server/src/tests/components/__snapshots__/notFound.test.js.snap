// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorNotFound component unit tests should render ErrorNotFound correctly 1`] = `
<div>
   The requested page couldn't be found try with another page
</div>
`;
